src/main/resources/db/changelog/v.0.3/db.changelog-skill.yaml

```diff
@@ -0,0 +1,64 @@
+databaseChangeLog:
+  - changeSet:
+      id: 12042021-add-type-skills
+      author: maksim_maevsky
+      dbms: postgresql
+      sql: CREATE TYPE SKILL as ENUM ('JAVA', 'JS', 'C_SHARP', 'C_PLUS_PLUS', 'QA', 'DEV_OPS', 'SWIFT', 'GO', 'PHP', 'PYTHON', 'KOTLIN', 'RUST');
+
+  - changeSet:
+      id: 12042021-add-type-internship-skill
+      author: maksim_maevsky
+      changes:
+        - createTable:
+            tableName: internship_skill
+            columns:
+              - column:
+                  name: is_name
+                  type: SKILL
+                  constraints:
+                    nullable: false
+              - column:
+                  name: is_inship_id
+                  type: UUID
+                  constraints:
+                    nullable: false
+  - changeSet:
+      id: 12042021-add-foreignKeyConstraint-internship-to-internship_skill
+      author:  maksim_maevsky
+      changes:
+        - addForeignKeyConstraint:
+            baseColumnNames: is_inship_id
+            baseTableName: internship_skill
+            constraintName: fk_internship_skill
+            referencedColumnNames: inship_id
+            referencedTableName: internship
+
+  - changeSet:
+      id: 12042021-add-type-user-skill
+      author: maksim_maevsky
+      changes:
+        - createTable:
+            tableName: user_skill
```

Comment:
Please, add to both table uniqie constraint to avoid set the same skill to one user/internship.

Reply:
done
![image](https://user-images.githubusercontent.com/65280608/114505098-8ba6d400-9c38-11eb-9d7d-205cdf83c46b.png)